go.sum

```diff
@@ -148,3 +148,5 @@ gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7/go.mod h1:JAlM8MvJe8wmxCU4Bl
 honnef.co/go/tools v0.0.0-20190102054323-c2f93a96b099/go.mod h1:rf3lG4BRIbNafJWhAfAdb/ePZxsR/4RtNHQocxwk9r4=
 pack.ag/amqp v0.11.0 h1:ot/IA0enDkt4/c8xfbCO7AZzjM4bHys/UffnFmnHUnU=
 pack.ag/amqp v0.11.0/go.mod h1:4/cbmt4EJXSKlG6LCfWHoqmN0uFdy5i/+YFz+fTfhV4=
```

Comment:
If you bump azure-amqp-common-go to v2.1.0, you will only have amqp v0.11.2 rather than both v0.11.0 and v0.11.2 in the go.sum.

Here are the changes between the version 2.1.0 and 2.0.0 https://github.com/Azure/azure-amqp-common-go/compare/v2.0.0...v2.1.0